src/lib/pages/frontsite/Literature/LiteratureSearch/LiteratureSearch.js

```diff
@@ -85,6 +86,7 @@ class LiteratureSearch extends Component {
                       {...invenioConfig.APP.SEARCH_BAR_PROPS}
                       responsiveAutofocus
                     />
+                    <SearchGuideLink />
```

Comment:
Is it added also to the literature details page? Or only to the search page?

Reply:
Currently it's only added to the search page, not to the details. I'm not sure if it should be added to the details, but I could argue that if a user reaches a details page it's already more on track of what to search.